epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -125,6 +149,18 @@ public TypeName getParameterizedGeneratedName() {
     return parameterizedClassName;
   }
 
+  private void removeMethodIfDuplicatedBySetter(Collection<AttributeInfo> attributeInfos) {
+    for (AttributeInfo attributeInfo : attributeInfos) {
+      Iterator<MethodInfo> iterator = methodsReturningClassType.iterator();
+      while (iterator.hasNext()) {
+        MethodInfo methodInfo = iterator.next();
+        if (methodInfo.name.equals(attributeInfo.getName())) {
```

Comment:
Thanks! I haven't thought about it, but it totally makes sense. Definitely will fix it.